x/distribution/client/cli/tx.go

```diff
@@ -224,16 +237,14 @@ $ %s tx distribution fund-community-pool 100uatom --from mykey
 				return err
 			}
 
-			return tx.GenerateOrBroadcastTx(clientCtx, msg)
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
 		},
 	}
-
-	return cmd
 }
 
 // GetCmdSubmitProposal implements the command to submit a community-pool-spend proposal
 func GetCmdSubmitProposal(clientCtx client.Context) *cobra.Command {
```

Comment:
Should I remove the `client.Context` arg here as part of #6664 ?

Reply:
Yes, good catch. I removed it in this PR as well.

Reply:
Ah perfect, thanks, less work for me 😄